src/pages/PostsList.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <q-page class="page-post-list flex justify-center">
+    <div class="page-post-list__table-list full-width q-px-xl">
+      <div class="bg-info page-post-header row justify-between">
+        <div class="text-h1 q-my-lg">Blog</div>
+        <q-btn rounded color="primary" label="Novo Post" to="createAuthor" class="self-end"/>
+      </div>
+      <div class="q-my-lg">
+        <q-list class="q-gutter-y-md q-mt-sm">
+          <div class="page-post-list__card row rounded-borders bg-warning shadow-up-2">
+            <div class="page-post-list__card__image col-12 col-md-6 bg-primary">
+              <div class="page-post-list__category-tag">Category</div>
+              <q-img
+                :src="image"
+                :ratio="16/9"
+              />
+            </div>
+            <div class="col-12 col-md-6 column justify-between bg-secondary">
+                <!-- Title -->
+                <div class="page-post-list__card__texts bg-accent column col-9 q-pa-sm">
+                  <div class="text-h4 page-post-list__card__texts-formation ellipsis col-3 bg-red">
+                    Títulooooooooooooooooooooooooooooooooooooooooooooooooo
+                  </div>
+                  <!-- Description -->
+                  <div class="page-post-list__card__texts-formation flex justify-center items-center col-7 q-pb-xl">
+                    Breve descriçãoooooooooooooooooooooo
+                  </div>
+                  <!-- Data and Author -->
+                  <div class="bg-positive row justify-between items-center col-2">
+                    <div class="page-post-list__card__texts-formation q-pl-md text-h6 col-9">
+                      Autooor
+                    </div>
+                    <div class="page-post-list__card__texts-formation q-pr-md col-3">
+                      Dataa
+                    </div>
+                  </div>
+                </div>
+                <!-- Button -->
+                <div class="bg-dark row justify-end">
+                  <a :href="href" class="page-post-list__card__link">
```

Comment:
- classe fora do padrão BEM
- lembre-se sobre o que comentamos sobre o `<a>` em volta de um `btn`. Há a prop `to` pra isso